transforms/images/polus-ftl-label-plugin/README.md

```diff
@@ -50,10 +58,49 @@ This plugin takes one input argument and one output argument:
 | `--connectivity` | City block connectivity                               | Input  | number     |
 | `--outDir`       | Output collection                                     | Output | collection |
 
+## Example Code
+```Linux
+cd examples
+mkdir output
+basedir=$(basename ${PWD})
+docker run -v ${PWD}:/$basedir labshare/polus-ftl-label-plugin:0.3.10 \
+--inpDir /$basedir/"images/" \
+--outDir /$basedir/"output/" \
+--connectivity 1
+```
+
+## Viewing the results using Python
+```Python
```

Comment:
We should include a line about installing dependencies.

`pip install bfio==2.1.9`